### ISSUE TITLE
VoiceOver does not read exact number of items in the dialog

### ISSUE THREAD

**Issue by esporysz** [SAP/fundamental-ngx#12832]
#### Is this a bug, enhancement, or feature request?

bug

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)

angular: 18.2.4
fundamental: 0.53.1-rc.23

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;

1. Create dialog like in the example [dialog](https://sap.github.io/fundamental-ngx/#/core/dialog#form-dialog).
2. Turn on VoiceOver (tested with MacOs version 14.7) and open dialog.

VoiceOver should announce title of the dialog, exact number of items in the dialog and item where we currently have a focus. Currently VoiceOver does not announce exact number of items in the dialog. You can compare it with [example](https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/examples/dialog/#ex_label) how it should behaves.

It was consulted with accessibility expert.
